packages/driver/src/cypress/chai_jquery.coffee

```diff
@@ -24,7 +34,8 @@ $chaiJquery = (chai, chaiUtils, callbacks = {}) ->
       try
         ## always fail the assertion
         ## if we aren't a DOM like object
-        ctx.assert(false, args...)
+        ## depends on "negate" flag
+        ctx.assert(!!ctx.__flags.negate, args...)
```

Comment:
I think you actually may have fixed a much larger / bigger bug here where using negated assertions pass even when the DOM element doesn't exist.

I know I opened an issue and there are several others mentioning this... you should find them and see if this fixes them. If so - reference them all in this PR.